packages/open-payments/src/client/index.ts

```diff
@@ -41,6 +41,20 @@ export interface RouteDeps extends BaseDeps {
   logger: Logger
 }
 
+export interface UnauthenticatedResourceRequestArgs {
```

Comment:
it's a bit wordy given what this contains, but I like how the types read in the `grant.ts`, where the first method (to create a grant) has `UnauthenticatedResourceRequestArgs`, while all the other ones are `ResourceRequestArgs` with the explicit `accessToken`